MonoGame.Framework/Graphics/Effect/Resources/SpriteEffect.fx

```diff
@@ -40,7 +40,9 @@ VSOutput SpriteVertexShader(	float4 position	: POSITION0,
 
 float4 SpritePixelShader(VSOutput input) : SV_Target0
 {
-    return SAMPLE_TEXTURE(Texture, input.texCoord) * input.color;
+    float4 color = SAMPLE_TEXTURE(Texture, input.texCoord) * input.color;
+	color.rgb *= input.color.a;
+	return color;
 }
```

Comment:
I really don't think this is correct.  The existing code comes directly from the XNA implementation of the stock effects that you can see here:

http://xbox.create.msdn.com/en-US/education/catalog/sample/stock_effects